### ISSUE TITLE
Ability to select photos from folders for Albums 

### ISSUE THREAD

**Issue by AndyXheli** [nextcloud/photos#1511]
1. Create  a album
2. Add Photos
3. Only shows 
![image](https://user-images.githubusercontent.com/59488153/204381861-ac8b9704-4f68-4a45-86cd-424d83dd3b07.png)

There should be an option to browser folders so people can can add them that way.

**Comment by joskapista44** [comment]
When the new feature to organize your photos into albums became available, I was very excited, because I have thousands of photos of old events from the last 10-15 years organized in folders, and some events would be more efficient to access through albums.

However, when I tried to put in photos from a trip 3 years ago, I had to go through a very frustrating process:

    I had to scroll through thousands of images for about 20 minutes, during which time the images seemed to get stuck several times, having to be brought to life by scrolling up and down
    when I reached the desired images I noticed that hundreds of images were duplicated, as a shortened selection folder of the trip had been created for easier handling
    At this point I gave up, as it is impossible to filter this out in the current add picture view, and if I get something wrong I can re-scroll back 3 years, which is simply in the no way.

Solution suggestion:

    Reverse add logic, i.e. add images or folders directly
    If I have added a folder to an album, I can subsequently drop images from it one by one

**Comment by farhills** [comment]
I've landed in a similar situation.  It would be great if there was a multi-select tick box option to add/create albums in the 'browse by folder' part of the app.  The majority of my photos are auto-upload via phone in nested yyyy-mm folders, or manually organized in trip/event specific folders.  Would be nice to leverage the existing organization in folders. 

![image](https://user-images.githubusercontent.com/83388006/206018973-151e6783-20fd-4d17-b4d2-10cc1b09c9f6.png)

**Comment by burner-** [comment]
I totally agree. Currently whole album thing is just useless trash without that feature. We have 200GB of photos since 2009. We have stuck at old gallery app because currently there is no way to create new galleries with decent work from old images. That new photos had have regression at so many levels and it sad that after years of lauch it still does not have even basic features what it will need to be usable.

**Comment by W11T** [comment]
For my own album i use a workaround. Browse your photos and mark them as favourites. Then you have a smaller selection in your favourites view and can move it to the albums from there.
Unfortunatelly this doesn't work with collaborative albums. I found no way to add a foto from my favourites to a shared album when i am not the albums owner.
Could you possibly add such a feature?
Or - best - the requested option to directly add fotos from the album view using some sort of filter.

**Comment by defilor** [comment]
I would like to add to this open issue that creating albums from tags would be a really nice feature, spanning across shared files etc. 

It's really cumbersome to create even the simplest of albums the way file selections is implemented at this stage, which is quite a road block to using this feature altogether. 

Implementation of a right-click menu option to add folders or multiple selected files from "Files"-view should be a no-brainer from a user perspective as well. 

I hope this issue gets some more attention as creating and sharing albums would seem like a core feature of a collaborative cloud file hosting system.

**Comment by ados8** [comment]
Yes to this feature 🙏

**Comment by warioishere** [comment]
i would also like to habe this feature implemented. Or at least select photos not one by one. Maybe some can of click and hover to the last that has to be select. Its a real pain to select photos one by one and takes ages thus making it complete useless

**Comment by samwh1te** [comment]
Yes please to this!!!

**Comment by AndyXheli** [comment]
I started using the memories app is amazing 1000 times better then this app. Just works perfect and no issue. You still need the photos app enabled but I enabled the custom css  app and hid the photos app and the feedback from all the users was very positive for anyone that what's more off of photos all try memories app

**Comment by aimer63** [comment]
> I started using the memories app is amazing 1000 times better then this app. Just works perfect and no issue. You still need the photos app enabled but I enabled the custom css app and hid the photos app from the menu and the feedback from all the users was very positive for anyone that what's more off of photos all try memories app

Does it implements the requested feature?

**Comment by burner-** [comment]
Kinda it implements. You can do public share for folder and manually modify it to memories link so you will get public gallery. 
Example https://nc.vector.fi/index.php/apps/memories/s/wL3QTA48ZzcQo4R
That current photos is just rubbish at so many level. It is sad that old one got removed and replaced with this trash. Anyway memories return some usability.

**Comment by joskapista44** [comment]
why?

**Comment by warioishere** [comment]
@AndyXheli why do you close an issue that everyone wishes for, so many people hate the current foto application. You just close the issue without a comment? How, ehm impolite!

**Comment by AndyXheli** [comment]
Hi, I’m no longer using Nextcliue and don’t want to be updated on its issues. If you want to keep this issue open, please open a new ticket. Thanks!

**Comment by fwsmit** [comment]
I believe you can just unsubscribe without closing the issue

**Comment by Doomsdayrs** [comment]
Bump to this request, who maintains this repository?

**Comment by joshtrichards** [comment]
Isn't this the same as #1365?
